<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вставить - переместить</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="Text_window_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Пишите здесь</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Text_window_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить копию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Переместить </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>105</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Вставить html код</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="HTML_code_window">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>351</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HTML код для вставки:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>105</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Основной текст -&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>330</y>
      <width>240</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;- копия / перенос / результаты html кода</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
